samples/embedded-auth-with-sdk/src/test/java/info/seleniumcucumber/userStepDefinitions/Login.java

```diff
@@ -83,6 +83,16 @@ public void refresh_token_stored() {
 		Assert.assertFalse("Can't access refresh_token", refreshToken.isEmpty());
 	}
 
+	@And("^the cell for the value of \"email\" is shown and contains her email$")
+	public void the_cell_for_email_is_shown_and_contains_her_email() {
+		Assert.assertTrue(rootPage.email.isDisplayed());
```

Comment:
I think we should add some more validations here: We're missing the part `and contains her email`